client/lib/plans/constants.js

```diff
@@ -159,24 +159,9 @@ export const TYPE_PERSONAL = 'TYPE_PERSONAL';
 export const TYPE_PREMIUM = 'TYPE_PREMIUM';
 export const TYPE_BUSINESS = 'TYPE_BUSINESS';
 
-const WPComGetBillingTimeframe = abtest => {
-	if ( abtest ) {
-		if ( isEnabled( 'upgrades/2-year-plans' ) && abtest( 'multiyearSubscriptions' ) === 'show' ) {
-			return i18n.translate( '/month, billed annually or every two years' );
-		}
-	}
-	return i18n.translate( 'per month, billed yearly' );
-};
-
-const WPComGetBiennialBillingTimeframe = abtest => {
-	if ( abtest ) {
-		if ( isEnabled( 'upgrades/2-year-plans' ) && abtest( 'multiyearSubscriptions' ) === 'show' ) {
-			return i18n.translate( '/month, billed every two years' );
-		}
-	}
-
-	return WPComGetBillingTimeframe( abtest );
-};
+const WPComGetBillingTimeframe = () =>
```

Comment:
Is there a reason this needs to be a function? Looks like it could be a simple constant now. Of course, that will add a few more changed lines, but ultimately it will make the code simpler, I think.

Reply:
@klimeryk agreed, I'll batch it with some other janitorial changes